app/visualisation_methods/interaction/_modal.html.haml

```diff
@@ -116,6 +122,7 @@
       query_params += "&param_y="+y_axis;
       query_params += "&output="+outputParam;
       query_params += "&chart_id="+interaction_chart_counter;
+      query_params += "&stand_alone=false";
```

Comment:
Co prawda to już było wcześniej napisane, ale dlaczego konstruować query_params zamiast używać hasha z paramterami w getWithSession? Jest to bardziej czytelnie i bezpieczniejsze.